src/components/Tab/Tab.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { CSSProperties } from "react";
+import { motion } from "framer-motion";
+
+import * as S from "./Tab.styles";
+import { NAV_HEIGHT } from "@/styles/theme";
+
+interface TabComponentProps {
+  tabContents: Array<string>;
+  tabClickHandler: (index: number) => void;
+  focusedTab: number;
+  style?: CSSProperties;
+  width: string | number;
+  height: string | number;
+}
+
+const Tab: React.FC<TabComponentProps> = ({
+  tabContents,
+  tabClickHandler,
+  focusedTab = -1,
+  width = "100%",
+  height = `${NAV_HEIGHT}rem`,
+  style,
+}) => {
```

Comment:
[ P3 ] 특별한 사유가 없으시다면!
 요렇게 수정하시면 컨벤션에도 맞을것 같습니다! 😀

```suggestion
const Tab = ({
  tabContents,
  tabClickHandler,
  focusedTab = -1,
  width = "100%",
  height = `${NAV_HEIGHT}rem`,
  style,
} : TabComponentProps) => {
```

Reply:
네! 레거시 코드 긁어오다가 이부분은 수정안했던것 같네요